owa/app/i18n/messages.js

```diff
@@ -7,26 +7,26 @@ export default {
         BED_TAGS: 'Bed Tags',
         LOGOUT: 'Logout',
         CHOOSE_LANGUAGE: 'Choose language',
-        ROW_REQUIRED_MSG: 'Row is required field',
-        COLUMN_REQUIRED_MSG: 'Column is required field',
+        ROW_REQUIRED_MSG: 'Row is a required field',
+        COLUMN_REQUIRED_MSG: 'Column is a required field',
         ROW_SHOULD_GREATER_THAN_ZERO: 'Row value should be greater than 0',
         ROW_SHOULD_GREATER_THAN_LAYOUT_ROW: 'Row value should not be greater than layout row size',
         COLUMN_SHOULD_GREATER_THAN_ZERO: 'Column value should be greater than 0 and less than 11',
-        COLUMN_SHOULD_GREATER_THAN_LAYOUT_COLUMN: 'Row value should not be greater than layout row size',
+        COLUMN_SHOULD_GREATER_THAN_LAYOUT_COLUMN: 'Column value should not be greater than layout column size',
         FIX_ERROR_MSG: 'Fix error before submit',
-        ADMISSION_LOCATION_SAVE_MSG: 'Admission location save successfully',
+        ADMISSION_LOCATION_SAVE_MSG: 'Admission location saved successfully',
         BED_TYPES_NOT_CONFIGURED_MSG: 'Bed Types must be configured before you can add a bed',
-        BED_LAYOUT_SAVE_MSG: 'Admission location bed layout save successfully',
-        BED_TAG_SAVE_MSG: 'Bed Tag save successfully',
-        BED_TYPE_SAVE_MSG: 'Bed Type save successfully',
-        BED_SAVE_MSG: 'Bed save successfully',
+        BED_LAYOUT_SAVE_MSG: 'Admission location bed layout saved successfully',
+        BED_TAG_SAVE_MSG: 'Bed Tag saved successfully',
```

Comment:
Strictly speaking in English there is no reason to use title case here then, so let's do:
```
BED_TAG_SAVE_MSG: 'Bed tag saved successfully',
```